src/ai/backend/common/events.py

```diff
@@ -529,6 +530,21 @@ def __call__(
 ]
 
 
+class KernelLifecycleEventReason(str, enum.Enum):
+    ALREADY_TERMINATED = "already-terminated"
+    FAILED_TO_START = "failed-to-start"
+    FORCE_TERMINATED = "force-terminated"
+    KILLED_BY_EVENT = "killed-by-event"
+    PENDING_TIMEOUT = "pending-timeout"
+    SELF_TERMINATED = "self-terminated"
+    TASK_DONE = "task-done"
+    TASK_FAILED = "task-failed"
+    TASK_TIMEOUT = "task-timeout"
+    TASK_CANCELLED = "task-cancelled"
+    TASK_FINISHED = "task-finished"
+    USER_REQUESTED = "user-requested"
+
```

Comment:
Let's add new `UNKNOWN` value to be used as default value of `reason`.
```suggestion
    UNKNOWN = "unknown"
```